caluma/extensions/permissions.py

```diff
@@ -16,6 +16,9 @@ def _is_admin_or_sagw(self, info):
     def _is_own(self, info, instance):
         return instance.created_by_user == info.context.user.username
 
+    def _is_assigned(self, info, instance):
+        return info.context.user.claims["sub"] in instance.assigned_users
```

Comment:
Why do we use the username in `_is_own` and the "sub" claim here?

Reply:
was `username` incorrect? If the two properties are identical, i'd find `username` more readable.

Reply:
Checking the username is incorrect here

Reply:
ok :+1: